### ISSUE TITLE
Skip digger if digger.yml if absent

### ISSUE THREAD

**Issue by motatoes** [diggerhq/digger#1177]
current behaviour if digger.yml is absent in a repo PR is that digger will try to automatically traverse folders and generate projects itself.

This behaviour should be changed to skip digger entirely and Noop, no comments left on PR. This is to avoid random failure messages appearing in a repo before it is actually setup

If a user wants digger to automatically discover and traverse projects the behaviour should be explicitly configured using `generate_projects` or others specified in a digger.yml

Reported by NL

**Comment by thiagowfx** [comment]
Indeed. When we added Digger for the first time to a large existing repo, it emitted failure comments in unrelated PRs, which is noisy:

<img width="916" height="167" alt="Image" src="https://github.com/user-attachments/assets/ff8408c2-bcd1-43d6-8a0f-bdf8e8c035c9" />
